mappings/net/minecraft/fluid/FluidState.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_3610 net/minecraft/fluid/FluidState
-	METHOD method_15756 (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
+	METHOD method_15756 isVisibleFromASide (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
```

Comment:
`isSideVisible`/`isVisibleFromSide`

Reply:
@Runemoro note the parameters are a block view and a block pos, so I doubt the name.

Reply:
Yeah, not sure where side is coming from. I will need to look at the code.

Reply:
I think I'll unname this for now